autobahn/twisted/resource.py

```diff
@@ -34,6 +34,7 @@
     # starting from Twisted 12.2, NoResource has moved
     from twisted.web.resource import NoResource
 from twisted.web.resource import IResource, Resource
+from twisted.python.compat import _PY3
```

Comment:
This looks like a private API from twisted. why not using six?

This breaks with twisted < 12. see https://github.com/buildbot/buildbot/pull/1808